controllers/argocd/keycloak_test.go

```diff
@@ -208,6 +208,7 @@ func TestNewKeycloakTemplate_testRoute(t *testing.T) {
 }
 
 func TestKeycloak_testRealmConfigCreation(t *testing.T) {
+	templateAPIFound = false
```

Comment:
Is this change intention? 
BTW, I would love to get rid of this `templateAPIFound` business.

Reply:
removed this. Thanks :)